src/sentry/utils/performance_issues/detectors/io_main_thread_detector.py

```diff
@@ -30,23 +32,29 @@
 
 
 class BaseIOMainThreadDetector(PerformanceDetector):
-    __slots__ = ("spans_involved", "stored_problems")
+    __slots__ = ("stored_problems",)
```

Comment:
the `__slots__` on all of these classes are wrong -- plus the baseclass does not have `__slots__` so this doesn't do what one would usually want with slots (removing `__dict__` and disallowing attribute stuffing).  I'm leaving it alone beyond removing the unused attributes but someone should fix this (probably by removing `__slots__` entirely, or by setting `__slots__ = ()` on the baseclass and then fixing all the incorrect `__slots__` assignments)